pkg/pipeline/params/params.go

```diff
@@ -561,7 +561,7 @@ func (p *Params) updateFilepath(identifier string) error {
 
 	if p.StorageFilepath == "" || strings.HasSuffix(p.StorageFilepath, "/") {
 		// generate filepath
-		p.StorageFilepath = fmt.Sprintf("%s%s-%s%s", p.StorageFilepath, identifier, time.Now().String(), ext)
+		p.StorageFilepath = fmt.Sprintf("%s%s-%d%s", p.StorageFilepath, identifier, time.Now().Unix(), ext)
```

Comment:
Unix() isn't very human friendly, can we do something like this?

`time.Now().Format("2006-01-02T150405")`